iati/using_data/templates/using_data/using_data_page.html

```diff
@@ -29,6 +29,7 @@
                   <div class="card card--alt">
                       <h2 class="card__heading">{{ subpage.heading }}</h2>
                       <p class="card__excerpt">{{ subpage.excerpt|default:"" }}</p>
+                      <p class="card__excerpt--truncated">{{ subpage.excerpt|truncatechars:35|default:"" }}</p>
```

Comment:
Here, if the subpage has no `excerpt` data, 'None' appears when the breakpoint is hit. I'm not sure if this is something to do with how `truncatechars` handles null values, might be worth wrapping this in a conditional instead of using `default` as the fallback.

<img width="1012" alt="Screenshot 2019-04-30 at 15 16 32" src="https://user-images.githubusercontent.com/8557782/56968372-f7cac300-6b5a-11e9-998b-e5e57b3d63f8.png">

Reply:
I think we can instead shuffle `truncatechars` and `default` around, so it would print an empty string that gets truncated, rather than a `None` value that gets casted implicitly to string (which is I guess what happened here)

Reply:
Yes! That would be muuuuuuuch more logical!